gui/journal.lua

```diff
@@ -98,13 +171,20 @@ function TextEditor:updateScrollbar()
 end
 
 function TextEditor:renderSubviews(dc)
-    self:updateScrollbar()
-
     self.editor.frame_body.y1 = self.frame_body.y1-(self.render_start_line_y - 1)
     self.editor:render(dc)
     self.scrollbar:render(dc)
 end
 
+function TextEditor:onInput(keys)
+    if (self.scrollbar.is_dragging) then
+        return self.scrollbar:onInput(keys)
+    end
+
+    return self:inputToSubviews(keys)
```

Comment:
is is more common to write this as `return TextEditor.super.onInput(self, keys)`